Gallery with tag function

Yewon000 opened the pull request:
Added function
load tags
insert new tag
delete tag

Yewon000 committed:
Auto stash before merge of "Yewon" and "origin/develop"

Yewon000 committed:
Gallery with tag function

load tags
insert tag
delete tag

Yewon000 committed:
Merge branch 'Yewon' into origin/develop

Yewon000 committed:
Gallery update

zoom, rotate function is added

Yewon000 committed:
Gallery - add opensource

opensource

vkdnjdldnjsw reviewed (changes requested):
1. taglist's height is fixed, so some smartphone can't see the addTagBtn .
2. the addTagBtn's height is match_parent now, but i think it's more good to wrap_content.
3. please follow camel naming. so you need to change taglist to tagList.
4. (done)you need to move tagListAdapter.notifyDataSetChanged(); to below of tagListItems.add(new TagListItem(temp));.

vkdnjdldnjsw committed:
move tagListAdapter.notifyDataSetChanged();

vkdnjdldnjsw committed:
fix some bug and name

taglist's height is fixed, so some smartphone can't see the addTagBtn .
the addTagBtn's height is match_parent now, but i think it's more good to wrap_content.
please follow camel naming. so you need to change taglist to tagList.

vkdnjdldnjsw committed:
Merge remote-tracking branch 'origin/develop' into Yewon

vkdnjdldnjsw reviewed (approved):
good
